.github/workflows/ci.yml

```diff
@@ -20,5 +20,5 @@ jobs:
     name: Puppet
     uses: voxpupuli/gha-puppet/.github/workflows/beaker.yml@v2
     with:
-      beaker_facter: 'zabbix_version:Zabbix:5.0,6.0,7.0'
+      beaker_facter: 'zabbix_version:Zabbix:5.0,6.0'
```

Comment:
I've realized I'm missing something when I add zabbix 7.0 to the CI, can you fix this in this PR?

Reply:
fixed it